docs/maintainers/generic-how-to-implement-a-concept-exercise.md

```diff
@@ -28,41 +29,67 @@ languages
         └── concept
             └── &lt;SLUG&gt;
                 ├── .docs
+                |   ├── after.md
+                |   ├── hints.md
                 |   ├── instructions.md
                 |   ├── introduction.md
-                |   ├── hints.md
-                |   ├── source.md (required if there are third-party sources)
-                |   └── after.md
+                |   └── source.md (required if there are third-party sources)
```

Comment:
The order doesn't match the track-specific files. The latter were reordered, so for consistency they should match.

Reply:
Done!